mcs/class/System/Test/System.Net.Security/SslStreamTest.cs

```diff
@@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
 		}
 	}
 
+	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
+	[Test()]
+	public void SelfCompatibilityTest()
+	{
+		#if __MonoCS__
```

Comment:
I may have posted this in the wrong place before - here it is again - The issue that requires the #if MonoCS is not detection of execution platform - the code needs to compile without Mono.Security reference, when compiled for .Net. This was discussed on the mono-dev list http://lists.ximian.com/pipermail/mono-devel-list/2014-December/042533.html